andela/healthchecks-invincibles
# #150798557 Build a help center for videos and chats

#### What does this PR do?
It is a help center where one can view FAQs, watch videos and have a live chat.
#### Description of Task to be completed?
A user should find a myriad of tutorials, help videos, live chat, FAQ's that will ease the  process of getting onboard and help answer some common questions that may arise.
#### How should this be manually tested?
* After cloning the repo, cd into it and run `./manage.py runserver`
* Then go to the following `urls`:
     `http://127.0.0.1:8000/videos/faqs/`- To see the FAQs questions
     `http://127.0.0.1:8000/videos/videos/`- To watch help videos
* At the bottom right side of your machine you will find a chat center where you can easily sent a text and receive a respond
#### What are the relevant pivotal tracker stories?
#150798557
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28527699/30658369-2916f956-9e43-11e7-88b5-5635e01b82a7.png)
